examples/access-management/version.tf

```diff
@@ -1,5 +1,5 @@
 terraform {
-  required_version = ">= 1.3.0, <1.7.0"
+  required_version = ">= 1.3.0, <2.0.0"
```

Comment:
Remove the upper constraint all together - thats what we have done in all other repos